src/main/java/io/confluent/examples/streams/GlobalKTablesExample.java

```diff
@@ -152,19 +157,24 @@ public static KafkaStreams createStreams(final String bootstrapServers,
     final StreamsBuilder builder = new StreamsBuilder();
 
     // Get the stream of orders
-    final KStream<Long, Order> ordersStream = builder.stream(Serdes.Long(), orderSerde, ORDER_TOPIC);
+    final KStream<Long, Order> ordersStream = builder.stream(ORDER_TOPIC, Consumed.with(Serdes.Long(), orderSerde));
 
     // Create a global table for customers. The data from this global table
     // will be fully replicated on each instance of this application.
     final GlobalKTable<Long, Customer>
         customers =
-        builder.globalTable(Serdes.Long(), customerSerde, null, CUSTOMER_TOPIC, CUSTOMER_STORE);
+        builder.globalTable(CUSTOMER_TOPIC, Materialized.<Long, Customer, KeyValueStore<Bytes, byte[]>>as(CUSTOMER_STORE)
+                .withKeySerde(Serdes.Long())
+                .withValueSerde(customerSerde));
+
 
     // Create a global table for products. The data from this global table
     // will be fully replicated on each instance of this application.
     final GlobalKTable<Long, Product>
         products =
-        builder.globalTable(Serdes.Long(), productSerde, null, PRODUCT_TOPIC, PRODUCT_STORE);
+        builder.globalTable(PRODUCT_TOPIC, Materialized.<Long, Product, KeyValueStore<Bytes, byte[]>>as(PRODUCT_STORE)
```

Comment:
Nit: fix indention -- we are using 2 spaces in examples, not 4

Reply:
there is a mixture of 4 and 2 spaces everywhere. I think we need to all have the same style settings in our IDE